lottie/build.gradle

```diff
@@ -41,7 +41,6 @@ dependencies {
 
   annotationProcessor libs.nullaway
   errorprone libs.errorprone.core
-  errorproneJavac libs.errorprone.javac
```

Comment:
AFAIU this was needed only when running on Java 8

Reply:
I had ambitions to add a bunch of errorprone checks to the core lottie library but never got around to adding them.